joinfaces-autoconfigure/src/main/java/org/joinfaces/autoconfigure/servlet/initializer/JsfClassFactory.java

```diff
@@ -126,7 +126,9 @@ private boolean isClassesFolder(String file) {
 			}
 
 			// add project classes folder
-			for (URL url : ClasspathHelper.forManifest()) {
+			Collection<URL> urls = ClasspathHelper.forJavaClassPath().size() > ClasspathHelper.forManifest().size()
```

Comment:
How about merging the two collections instead of using the one or the other?

Reply:
Overview: 
- java 8: `forManifest()` method was default. `forManifest()` usually returns collection of urls greater than `forClasspath()` one when testing / debugging / running uber jar and running uber war.
- java 9: `forManifest()` returns empty collection when testing and debugging joinfaces. That's why java 9 build was broken. `forJavaClassPath()` returns a greater collection of urls that contains classes folder in these cases. When running ubber jar/war with joinfaces, `forManifest()` is used again.

I think merging two collections works too. What is the advantage of merging? The current implementation is a bit faster than merging.

Reply:
My idea was, that merging would give the best possible result in every situation.

Reply:
You are right.

Reply:
Done @larsgrefer